packages/manager/apps/dedicated/client/app/billing/confirmTerminate/legacy/termination-legacy.service.js

```diff
@@ -0,0 +1,66 @@
+export default /* @ngInject */ function BillingTerminate($q, OvhHttp) {
```

Comment:
Didn't update the code (as its legacy) only changed this part to export and inject dependencies